src/python/dxpy/utils/executable_unbuilder.py

```diff
@@ -101,6 +101,13 @@ def _dump_app_or_applet(executable, omit_resources=False, describe_output=[]):
     with open(script, "w") as f:
         f.write(info["runSpec"]["code"])
 
+    # Used to unpack cluster bootstrapScript(s) if they exist for this app(let)
+    def make_cluster_bootstrap_script_file(region, entry_point, code, suffix):
```

Comment:
Our convention is to use triple double quotes for methods comments/descriptions. Thanks!